### PR TITLE
add evict command

### PR DESCRIPTION
**Pull request by Mojachieee**
Implements a rudimentary evict replica command

### CODE REVIEW

File: cmd/evict/evict.go [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+	if err != nil {
+		return err
+	}
+
+	// Do some basic sanity checking to ensure we're not about to destroy a volume
+	if len(vol.Replicas) < 1 {
+		return fmt.Errorf("cannot evict volume if it has no replicas - data would be lost")
+	}
+
+	if vol.Master.Health != health.MasterOnline {
```
Comment (Ricardo-Osorio) [inline cmd/evict/evict.go:136]: Out of curiosity, wouldn't this be something you expect to be done server side?

File: cmd/evict/evict.go [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+			}
+		}
+	}
+
+	if deploymentToRemove == "" {
+		return errors.New("failed to find deployment on target node for volume")
+	}
+
+	if c.cordonTargetNode {
+		fmt.Println("cordining node")
```
Comment (Ricardo-Osorio) [inline cmd/evict/evict.go:163]: Are these debug statements?
Also, typo on the word "cordon"

Comment (Mojachieee) [reply cmd/evict/evict.go]: Nope, they're meant for user output, I didn't want to setup a logger for this relatively simple command (most over commands also use `fmt`.

File: cmd/evict/evict.go (hunk 1 above)
Comment (Mojachieee) [reply cmd/evict/evict.go:137]: Yeah the control plane will just barf if we attempt to do this with an unhealthy master. That being said it seems sensible to also check here, as it means we can print a nice error

**Review by Ricardo-Osorio — APPROVED** [review]

**Pull request merged**